assets/js/blocks/price-filter/block.js

```diff
@@ -26,10 +27,12 @@ import usePriceConstraints from './use-price-constraints.js';
  */
 const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 	const [ minPriceQuery, setMinPriceQuery ] = useQueryStateByKey(
```

Comment:
`useQueryStateByKey()` sets the default value to an empty object, I added the `null` parameter so the default value is `null`, this avoids unnecessary re-calculations when using `minPriceQuery` and `maxPriceQuery` in hook dependencies.